### ISSUE TITLE
Add remaining statistics and statistical plots

### ISSUE THREAD

**Issue by AndreiKingsley** [Kotlin/kandy#369]

**Comment by devcrocod** [comment]
What is this issue about?
which statistics, plots?

**Comment by AndreiKingsley** [comment]
I’ll add more descriptions to this issue later, need to take all available statistics and plots from Let’s Plot

**Comment by AndreiKingsley** [comment]
There should be checklist with separated issues for each statistic.

**Comment by nuclear718** [comment]
> What is this issue about? which statistics, plots?

In the official documentation: https://kotlin.github.io/kandy/kandy-with-gradle.html

Regarding these two configurations provided in the official documentation:

```
repositories {
maven "https://packages.jetbrains.team/maven/p/kds/kotlin-ds-maven"
}
dependencies {
implementation "org.jetbrains.kotlinx:kotlin-statistics-jvm:0.1.0"
}
```

How many of you have actually tried it? After making these two configurations in Gradle within IntelliJ IDEA, it is simply not possible to run!

**Comment by AndreiKingsley** [comment]
@nuclear718 yes indeed the Groovy option is incorrectly written, thank you! The correct way to write it would be
```
repositories {
    maven {
        url 'https://packages.jetbrains.team/maven/p/kds/kotlin-ds-maven'
    }
}
```

**Comment by devcrocod** [comment]
@nuclear718 
We tested on kotlin scripting since this is now the main interaction with the gradle build script. You can use the [migration guide](https://docs.gradle.org/current/userguide/migrating_from_groovy_to_kotlin_dsl.html).

This problem is not related to this issue, please create a new one if you are not sure

**Comment by nuclear718** [comment]
> @nuclear718 yes indeed the Groovy option is incorrectly written, thank you! The correct way to write it would be
> 
> ```
> repositories {
>     maven {
>         url 'https://packages.jetbrains.team/maven/p/kds/kotlin-ds-maven'
>     }
> }
> ```
> 
> Created an issue #374

Thank you so much!

I follow your commented and it is work! My another issue(wrong closed by devcrocod) was solved too!!

thanks!thanks!thanks!thanks!thanks!thanks!thanks!thanks!thanks!thanks!thanks!thanks!

**Comment by devcrocod** [comment]
@AndreiKingsley What is the status of this issue?
